src/raftstore/store/store.rs

```diff
@@ -1634,11 +1649,11 @@ impl<T: Transport, C: PdClient> Store<T, C> {
 
     fn store_heartbeat_pd(&mut self) {
         let mut stats = StoreStats::new();
-        let disk_stats = match fs2::statvfs(self.engine.path()) {
+        let disk_stats = match fs2::statvfs(self.raft_engine.path()) {
```

Comment:
we need check if raft and kv are on same disk first.
if they are on differcnt disk,  since raft used_size is alway small (available is alway large),  we should use kv_engine to calc available?  or change pd to distinguish raft and kv engine?